main.yml

```diff
@@ -696,14 +699,22 @@ streams:
       - id: kafkas
         title: "Kafka Instances"
         default: true
-      - id: service-accounts
-        title: "Service Accounts"
       - id: resources
         title: "Resources"
       - id: streams-docs
         title: Documentation
         navigate: https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/
 
+service-registry:
+  title: "Service Registry"
+  frontend:
+    sub_apps:
+      - id: sr
+        title: "Service Registry"
+      - id: streams-docs
+        title: Documentation
+        navigate: https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/
```

Comment:
This should be updated to provide documentation for Service Registry. It still shows the URL and id for Kafka.

Reply:
BTW, about the 404, according to the CCS team, the contents are not live yet, but this should be the correct URL: 
https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry

Reply:
I have updated!
Thanks for the review